src/GovUk.Education.ExploreEducationStatistics.Content.Model/User.cs

```diff
@@ -12,6 +13,10 @@ public class User
 
         public string Email { get; set; }
 
+        public Guid DeletedById { get; set; }
+
+        public DateTime? SoftDeleted { get; set; }
```

Comment:
I think that with the recent changes renaming `Deleted` to `SoftDeleted` the database migration needs updating.